### PR TITLE
Bugfix/complete signup

### PR DESCRIPTION
**Pull request by Shreyaschorge**
Fixes #
- Horizontal Scroll
- Address Dropdown Alignment for geocoder
- Replace custom styled `MaterialTextField` with `TextField` from MUI

### CODE REVIEW

**Comment by mohitb35** [comment]
@Shreyaschorge Noticed some UI issues relating to responsiveness of the form, spacing, and padding.

Screenshot 2022-08-18 at 2 49 10 PM
Screenshot 2022-08-18 at 2 48 57 PM
Screenshot 2022-08-18 at 2 50 18 PM
Screenshot 2022-08-18 at 2 50 09 PM
Screenshot 2022-08-18 at 2 49 21 PM
Screenshot 2022-08-18 at 2 48 49 PM

**Review by mohitb35 — COMMENTED** [review]
@Shreyaschorge Noticed some UI issues relating to responsiveness of the form, spacing, and padding.

Screenshot 2022-08-18 at 2 49 10 PM
Screenshot 2022-08-18 at 2 48 57 PM
Screenshot 2022-08-18 at 2 50 18 PM
Screenshot 2022-08-18 at 2 50 09 PM
Screenshot 2022-08-18 at 2 49 21 PM
Screenshot 2022-08-18 at 2 48 49 PM

**Comment by mohitb35** [comment]
Points 1, 3 can be addressed later - these have been added to https://github.com/Plant-for-the-Planet-org/planet-webapp/issues/1557

> @Shreyaschorge Noticed some UI issues relating to responsiveness of the form, spacing, and padding.
> 
> 1. The field widths on mobile vary differently for autocomplete field and other field.
> 2. On mobile, there is always a white space below the form (the form scrolls within a reduced area).
> 3. Spacing between fields appears to be inconsistent.
> 
> Screenshot 2022-08-18 at 2 49 10 PM Screenshot 2022-08-18 at 2 48 57 PM Screenshot 2022-08-18 at 2 50 18 PM Screenshot 2022-08-18 at 2 50 09 PM Screenshot 2022-08-18 at 2 49 21 PM Screenshot 2022-08-18 at 2 48 49 PM

File: src/features/user/CompleteSignup/CompleteSignup.module.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 // entire page
 .signUpPage {
-  min-height: 100vh;
+  min-height: 150vh;
```
Comment (mohitb35) [inline src/features/user/CompleteSignup/CompleteSignup.module.scss:5]: @Shreyaschorge I don't think this solution is advisable. Revert this change and let's handle it later.

Comment (mohitb35) [reply src/features/user/CompleteSignup/CompleteSignup.module.scss:5]: I've added this in https://github.com/Plant-for-the-Planet-org/planet-webapp/issues/1557

Comment (Shreyaschorge) [reply src/features/user/CompleteSignup/CompleteSignup.module.scss:5]: @mohitb35 The way the form is structured its the only way. Or else we need to change the entire CSS of that form.

Comment (mohitb35) [reply src/features/user/CompleteSignup/CompleteSignup.module.scss]: Yes, agreed. That's why it's better to leave that point open.

**Review by mohitb35 — APPROVED** [review]
Looks good to me

**Comment by mariahosfeld** [comment]
Looks good to me, remaining styling issues will be dealt with separately.

**Pull request merged**